pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -365,9 +365,15 @@ func (s *Service) doReload(ctx context.Context) {
 	}
 
 	for provider, connector := range s.reloadables {
-		setting := getSettingByProvider(provider, settingsList)
+		settings := getSettingByProvider(provider, settingsList)
```

Comment:
I replaced `setting` with `settings` because we had a name collision with the package `setting`.